hack/provider/provider.yaml

```diff
@@ -18,6 +19,10 @@ optionGroups:
       - INJECT_GIT_CREDENTIALS
     name: "Agent options"
     defaultVisible: false
+  - options:
+      - 
```

Comment:
We could add AZURE_CUSTOM_DATA to the advanced options

Reply:
Sure, thought about that too